pytest_client_tools/plugin.py

```diff
@@ -256,3 +263,12 @@ def pytest_runtest_logfinish(nodeid, location):
     if node_running_data.logfile.exists():
         node_running_data.artifacts.copy(node_running_data.logfile)
     logging.getLogger().handlers.remove(node_running_data.handler)
+
+
+def pytest_runtest_setup(item):
+    envnames = [mark.args[0] for mark in item.iter_markers(name="env")]
+    the_environment = _settings.get("env_for_dynaconf") or "development"
```

Comment:
this is exactly the reason why the `TestConfig` class (and the `test_config` fixture) exists: group all the logic related to the "configuration" of tests in a single place; in this case, I'd not want to spread the logic that determines what is the current environment all over the place, but rather implement this once in `TestConfig` -- something like:
```python
class TestConfig:
    ...
    @property
    def environment(self):
        return _settings.get("env_for_dynaconf") or DEFAULT_ENVIRONMENT
```
this way:
- `DEFAULT_ENVIRONMENT` is used there, or even can be defined inline (without a variable) if there is no use for it outside of this function
- whatever needs to know what's the current environment can simply query `TestConfig.environment` without any logic for reading what's the existing one, and determining the fallback

Reply:
there is 'default_env' property in Dynaconf. It could be used to specify default environment.

Reply:
I wanted to avoid a class in this case - since Dynaconf does the same. Groupping functions into one namespace can be done by using module file and functions in it. It is easier for me to update doc aka 'this is a dynaconf object. See: dynaconf.org for more details how it works' rather that keep doc about test_class updated. Another reason was that when a test code is shallow (ie no additional middleware - it is easier to debug it and it is easier for an upstream user to understand the code'. It was my main reason for this suggestion - KISS.

Reply:
As I said, `TestConfig` does not replace dnyaconf is any way, but rather uses it underneath (composition). Even the documentation will still mention dynaconf, that is not going to change.

Thanks for the suggestion, but please use `TestConfig` and `test_config`.